frontend/src/pages/projects/types.ts

```diff
@@ -64,7 +64,7 @@ export type StartNotebookData = {
   image: ImageStreamAndVersion;
   volumes?: Volume[];
   volumeMounts?: VolumeMount[];
-  tolerationSettings?: NotebookTolerationSettings;
+  tolerationSettings?: TolerationSettings;
```

Comment:
Yeah, I thought of doing this change in #938 -- this will cause a conflict in naming.